omnibus_overrides.rb

```diff
@@ -1,5 +1,7 @@
 override :erlang, version: "18.3.4.9"
 override :lua, version: "5.1.5"
+override :rubygems, version: "2.7.7"
```

Comment:
You might be able to bump to 3.0 without issues here. It passed in Travis for chef/chef except for 1 spec where the output was a bit different.